src/lib/popover.ts

```diff
@@ -0,0 +1,8 @@
+export const POPOVER_ALIGN = {
```

Comment:
I would rather not put any display logic in `src/lib`. Could we put the module in `src/components` ? Either by creating a `Popover` directory (e.g. `Popover/index.vue` and `Popover/constants.vue`) or just by dropping it directly in `src/components`.